Header header

float32[4] velocity

float32[4] correlation

float32[4] intensity

float32[4] quality
